docs/language-basics/decorators.md

```diff
@@ -25,31 +25,31 @@ model Dog {
 }
 ```
 
-The parentheses can be omitted when no arguments are provided.
+If no arguments are provided, the parentheses can be omitted.
 
 ```typespec
 @mark
 model Dog {}
 ```
 
-## Augment decorators
+## Augmenting Decorators
```

Comment:
I think the title were better before, we are not augmenting decorators we are using an augment decorator